tests/test_base.py

```diff
@@ -46,9 +46,19 @@ def test_base_size(auto_container, container_runtime):
             "ppc64le": 160,
             "s390x": 125,
         }
-    assert (
+    container_size = (
         container_runtime.get_image_size(auto_container.image_url_or_id)
-        < BASE_CONTAINER_MAX_SIZE[LOCALHOST.system_info.arch] * 1024 * 1024
+        // 1024
+        // 1024
+    )
+    min_container_size = container_size - 3
+    assert (
+        min_container_size
+        < container_size
+        <= BASE_CONTAINER_MAX_SIZE[LOCALHOST.system_info.arch]
+    ), (
```

Comment:
I don't understand the logic behind this, won't `min_container_size < container_size` be always true?

Reply:
no. For tumbleweed the containers are currently about 20MB larger than they actually should be because dbus-1 etc is pulled into the container, which it doesn't belong there. I want to detect when we finally managed to get rid of the excessive dependencies again by having a failing test, and then lower the expected container size to the lower values again. 

without tihs change, we would not notice after the container bloat has been fixed that another bloat has been added. I think we do want to notice if the size of the containers change significantly (where significantly here is defined to be +/- 3MB)